shared/templates/create_sysctl.py

```diff
@@ -117,4 +130,4 @@ def generate(self, target, serviceinfo):
 
     def csv_format(self):
         return("CSV should contains lines of the format: " +
-               "sysctlvariable,sysctlvalue")
+               "sysctlvariable,sysctlvalue,<datatype>")
```

Comment:
Let's use `[,<datetype>]` in the help message - we already have this marker of optional options elsewhere.

Reply:
Updated.